Examples/destination_plugins/MixpanelDestination.swift

```diff
@@ -0,0 +1,302 @@
+        return event
+    }
+    
+    func screen(event: ScreenEvent) -> ScreenEvent? {
+        if settings?["consolidatedPageCalls"] as? Bool ?? false,
+           var payloadProps = event.properties?.dictionaryValue {
+            
+            let eventName = "Loaded a Screen"
+            if let name = event.name {
+                payloadProps["name"] = name
+            }
+            mixpanelTrack(eventName, properties: payloadProps)
+            analytics?.log(message: "Mixpanel track \(eventName) properties \(payloadProps)")
+        } else if settings?["trackAllPages"] as? Bool ?? false {
+            
+            var finalEventName = "Viewed Screen"
+            if let eventName = event.name {
+                finalEventName = "Viewed \(eventName) Screen"
+            }
+            
+            mixpanelTrack(finalEventName, properties: event.properties?.dictionaryValue)
+            analytics?.log(message: "Mixpanel track \(finalEventName) properties \(String(describing: event.properties?.dictionaryValue))")
+        } else if settings?["trackNamedPages"] as? Bool ?? false, let eventName = event.name {
+            let finalEventName = "Viewed \(eventName) Screen"
+            mixpanelTrack(finalEventName, properties: event.properties?.dictionaryValue)
+            analytics?.log(message: "Mixpanel track \(finalEventName) properties \(String(describing: event.properties?.dictionaryValue))")
+        } else if settings?["trackCategorizedPages"] as? Bool ?? false, let category = event.category {
+            let finalEventName = "Viewed \(category) Screen"
+            mixpanelTrack(finalEventName, properties: event.properties?.dictionaryValue)
+            analytics?.log(message: "Mixpanel track \(finalEventName) properties \(String(describing: event.properties?.dictionaryValue))")
+        }
+        
+        return event
+    }
+    
+    func group(event: GroupEvent) -> GroupEvent? {
+        
+        guard let groupID = event.groupId, !groupID.isEmpty,
+              let groupIdentifierProperties = settings?["groupIdentifierTraits"] as? [String] else {
+            return event
+        }
+        
+        // No need to continue if we don't have any properties
+        if groupIdentifierProperties.isEmpty {
+            return event
+        }
+        
+        for key in groupIdentifierProperties {
+            let nsGroupID = NSString(string: groupID)
+            
+            if let traits = event.traits?.dictionaryValue as? Properties {
+                if let group = traits[key] as? String {
+                    mixpanel?.getGroup(groupKey: group, groupID: nsGroupID).setOnce(properties: traits)
+                }
+            }
+            
+            mixpanel?.setGroup(groupKey: key, groupID: nsGroupID)
+            analytics?.log(message: "Mixpanel setGroup \(key) groupID \(groupID)")
+        }
+        
+        return event
+    }
+    
+    func alias(event: AliasEvent) -> AliasEvent? {
```

Comment:
we've explicitly disabled alias in analytics-swift, fyi.  not sure how critical this bit is to overall operation.

Reply:
Left for now... lets re-evaluate after discussions.